### ISSUE TITLE
About broadcast_buffers=False in train_net.py

### ISSUE THREAD

**Issue by zhangliliang** [facebookresearch/maskrcnn-benchmark#692]
## ❓ Questions and Help

In line 37-42 of  tools/train_net.py,
```
  if distributed:
        model = torch.nn.parallel.DistributedDataParallel(
            model, device_ids=[local_rank], output_device=local_rank,
            # this should be removed if we update BatchNorm stats
            broadcast_buffers=False,
        )
```
This codebase set `broadcast_buffers` to false when init DDP.
And the comment show that, 'this should be removed if we update BatchNorm stats'.

However, in my experiment, it could train sucessfully with fbnet that suing BN and train-from-stratch using `configs/e2e_faster_rcnn_fbnet.yaml`.

It make me confused about the influence of the `broadcast_buffers=False` in DDP init.

Could someone has some ideas about it?
Thanks.

**Comment by zimenglan-sysu-512** [comment]
hi @fmassa 
if use sync-bn, broadcast_buffers should be set to True?

**Comment by mehameha998** [comment]
@zimenglan-sysu-512 
Refer to [this](https://github.com/pytorch/pytorch/issues/22095), I guess if model in train mode, this flag should be set as True.

**Comment by Jerrypiglet** [comment]
> hi @fmassa
> if use sync-bn, broadcast_buffers should be set to True?

Can I know how did you end up setting ``broadcast_buffers`` with syn-bn?
